### ISSUE TITLE
LDAP Auth via Passport

### ISSUE THREAD

**Issue by ndom91** [thedevs-network/kutt#125]
Hey Guys,

Has anyone gotten LDAP Auth through Passport.js up and running?

If not I was going to take a crack at it when I get back home (visiting the parents over the holidays).

If anyone has it, can you post a gist or make pull request?

**Comment by poeti8** [comment]
No, I haven't done that.

**Comment by k-apc** [comment]
hi @ndom91 , did you get to do this? I'm interested in ldap  too.

Cheers,

**Comment by ndom91** [comment]
No, but if you're comfortable with it, you cuold swap Passport out for [`next-auth`](https://next-auth.js.org/) which supports LDAP auth with a little modification. Tutorial: https://next-auth.js.org/tutorials/ldap-auth-example

**Comment by k-apc** [comment]
I think it would need to put time I don't currently have, but out of curiosity... what do you mean with "little modification"?, hehe. I hours/minutes how much would it take to someone with very little npm experience?

**Comment by ndom91** [comment]
Haha, so by "little modification" I just mean by following the linked tutorial. It's a bit mroe work than simply installing some packages via npm, but its not that complicated either.

**Comment by k-apc** [comment]
sounds good. I'll give it a try. have you done it? does it work?

**Comment by ndom91** [comment]
Yeah im a core team member of `next-auth`, it does work :)

**Comment by juhannc** [comment]
Are there any updates regarding ldap/ad auth for kutt?

**Comment by avecheria** [comment]
I am also interested in the LDAP auth feature

**Comment by 1of16** [comment]
bump
still havent found a link shortener with a LDAP based user management :(
